numba/pythonapi.py

```diff
@@ -1581,3 +1582,39 @@ def string_from_constant_string(self, string):
         cstr = self.context.insert_const_string(self.module, string)
         sz = self.context.get_constant(types.intp, len(string))
         return self.string_from_string_and_size(cstr, sz)
+
+    def call_jit_code(self, func, sig, args):
+        """Calls into Numba jitted code and propagate error using the Python
+        calling convention.
+        """
+        builder = self.builder
+        cres = self.context.compile_subroutine(builder, func, sig)
+        got_retty = cres.signature.return_type
+        retty = sig.return_type
+        if got_retty != retty:
+            # This error indicates an error in *func* or the caller of this
+            # method.
+            raise errors.LoweringError(
+                f'mismatching signature {got_retty} != {retty}.\n'
```

Comment:
This is an f-string right? I am asking because they were introduced in 3.6 and so this code may not work on versions prior to that.

Reply:
as of 0.48, Python 3.6 is our minimum supported version

Reply:
:+1: